### ISSUE TITLE
Feature request: publish Lambda layers in Malaysia, Mexico, Thailand

### ISSUE THREAD

**Issue by leandrodamascena** [aws-powertools/powertools-lambda-typescript#3200]
### Use case

AWS recently launched new region:

Asia Pacific (Malaysia) - ap-southeast-5

https://aws.amazon.com/pt/blogs/aws/now-open-aws-asia-pacific-malaysia-region/

### Solution/User Experience

Deploy Lambda Layers to this new region.

### Acknowledgment

- [x] This feature request meets [Powertools for AWS Lambda (TypeScript) Tenets](https://docs.powertools.aws.dev/lambda/typescript/latest/#tenets)
- [x] Should this be considered in other Powertools for AWS Lambda languages? i.e. [Python](https://github.com/aws-powertools/powertools-lambda-python/), [Java](https://github.com/aws-powertools/powertools-lambda-java/), and [.NET](https://github.com/aws-powertools/powertools-lambda-dotnet/)

### Future readers

Please react with 👍 and your use case to help us understand customer demand.

**Comment by dreamorosi** [comment]
Adding the newer Mexico region to the issue (& renamed title).

We'll aim to release them at the next release which will be in 2 weeks.

**Comment by dreamorosi** [comment]
The regions are all bootstrapped in the beta stage, we still need to:
- bootstrap the prod stages
- verify that the Lambda layers are balanced in the new regons

**Comment by dreamorosi** [comment]
@am29d I see you have bootstrapped the regions but didn't add the new regions to the workflow matrix.

With today's release the layer version in the 3 new regions went out of sync again. Please balance the versions again and add open a PR as soon as you can - so they don't go out of sync again.

**Comment by am29d** [comment]
Yep, wanted to finish the work before noon, but then release was out, fix incoming.

**Comment by dreamorosi** [comment]
Ran the [SSM parameter script](https://github.com/aws-powertools/powertools-lambda-typescript/actions/runs/13522904825) & verified that the correct layer was published in the new regions.
